config/tests/multi-pipe-rechunk.yaml

```diff
@@ -0,0 +1,118 @@
+##########################################
+#
+##########################################
+---
+type: config
+# Logging level can be one of:
+# OFF, ERROR, WARN, INFO, DEBUG, DEBUG2 (case insensitive)
+# Note DEBUG and DEBUG2 require a build with (-DCMAKE_BUILD_TYPE=Debug)
+log_level: debug
+#    log_level: info
+cpu_affinity: [2,3,4,5,8,9,10,11]
+frame_arrival_period: 0.001
+
+buffer_depth: 4
+
+# data gen:
+num_frames: 20
+
+input_size: 4
+frame_size: 4
+
+rechunk_factor: 5
+
+# Pool
+main_pool:
+    kotekan_metadata_pool: chordMetadata
+    num_metadata_objects: 15 * buffer_depth
+
+# Buffers
+host_in_buffer:
+    kotekan_buffer: standard
+    num_frames: buffer_depth
+    # frame_size: 
+    metadata_pool: main_pool
+
+# Buffer used for signalling only, between the two GPU pipelines.
+host_mid_buffer:
+    kotekan_buffer: standard
+    num_frames: buffer_depth
+    frame_size: 0
+    metadata_pool: main_pool
+
+host_out_buffer:
+    kotekan_buffer: standard
+    num_frames: buffer_depth
+    frame_size: input_size * rechunk_factor
+    metadata_pool: main_pool
+
+gen_data:
+    kotekan_stage: testDataGen
+    type: const8
+    values: [0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07]
+    #    array_shape: [64, 16]
+    #dim_name: ["X", "Y"]
+    out_buf: host_in_buffer
+
+gpuA:
```

Comment:
Oh, that's a good idea!  I think we'll get a good feel for this once we start layout out the real pipelines - eg the upchannelization & FRB chain is going to have like 6-7 pipelines, one per upchan factor, because they're going to operate at somewhat different cadences.